aten/src/ATen/cudnn/cudnn-wrapper.h

```diff
@@ -5,9 +5,10 @@
 #define STRINGIFY(x) #x
 #define STRING(x) STRINGIFY(x)
 
-#if CUDNN_MAJOR < 6
-#pragma message ("CuDNN v" STRING(CUDNN_MAJOR) " found, but need at least CuDNN v6. You can get the latest version of CuDNN from https://developer.nvidia.com/cudnn or disable CuDNN with USE_CUDNN=0")
-#pragma message "We strongly encourage you to move to 6.0 and above."
+#if CUDNN_MAJOR < 8 || CUDNN_MINOR <= 5
```

Comment:
Wait shouldn't this be and && otherwise the could fit for CUDNN_MAJOR 9.0 too right?

Reply:
It should be #if CUDNN_MAJOR < 8 || CUDNN_MINOR < 5 since the negation is #if CUDNN_MAJOR >= 8 && CUDNN_MINOR >= 5

Reply:
I still don't follow. 

Let CUDDNN MAJOR = 9
CUDNN_MINOR=0
if false || CUDNN_MINOR < 5
if false || true
if true -> Prints the message

Prints the annoying message, what am I missing here?

Reply:
Yeah I think the second predicate is dependent so we can't write it with just two terms:
```
if major < 9
    if major == 8
        if minor < 5
            too old
        else
            OK
else
    OK
```